src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -73,8 +73,10 @@
         }
 
         .bcs-activity-feed-annotation-activity.bcs-is-focused {
+            margin-bottom: 1px;
```

Comment:
What is this overriding? Can we fix it at the source?

Reply:
The bottom border is removed when focused which causes a very slight shift. I didn't want to cause any problems with existing cards, but I can move it up.

Reply:
Rather than removing the border, can we make it transparent? That will resolve the issue for all cards, it sounds like.

Reply:
Using a transparent bottom border will leave a slight angle on the new focused left border, which is why I went with the margin instead.

Reply:
I see. Can we still solve the issue for all cards in another way, then? Should we put the margin in the shared `.bcs-is-focused` rule?

Reply:
Yes, I will do that